### PR TITLE
global: fix invenio_ext imports

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: requirements-devel.txt [hunk 1]
```diff
@@ -30,3 +30,5 @@
 
 -e git+git://github.com/inveniosoftware/invenio-base.git#egg=invenio-base
 -e git+git://github.com/inveniosoftware/invenio-upgrader.git#egg=invenio-upgrader
+-e git+git://github.com/inveniosoftware/invenio-ext.git#egg=invenio-ext
+-e git+git://github.com/inveniosoftware/invenio-ext.git#egg=invenio-ext
```
Comment (jirikuncar) [inline requirements-devel.txt]: double - one should be probably `invenio-utils` if needed

File: setup.py [hunk 2]
```diff
@@ -47,6 +47,7 @@
     'pytest_cov>=1.8.0,<2.0.0',
     'pytest_pep8>=1.0.6',
     'coverage>=3.7.1',
+    'invenio-ext>=0.1.1',
```
Comment (jirikuncar) [inline setup.py]: IMHO it should be in `requirements`

Comment (hachreak) [inline setup.py]: I wrongly put 2 times ext in requirements... but pages doesn't use utils.

Comment (jirikuncar) [inline setup.py]: it should be moved after line 39

**Comment by jirikuncar** [comment]
@hachreak rebase and :shipit:

File: setup.py [hunk 3]
```diff
@@ -35,9 +35,10 @@
 
 requirements = [
     'Flask>=0.10.1',
-    'six>=1.7.2',
     'invenio-base>=0.2.1',
+    'invenio-ext>=0.1.1',
```
Comment (jirikuncar) [inline setup.py]: `0.2.0`?

**Comment by jirikuncar** [comment]
@hachreak @SamiHiltunen will you replace also `invenio.testsuite`?

**Comment by hachreak** [comment]
done

**Pull request closed without merging**